main.tf

```diff
@@ -91,14 +91,18 @@ locals {
     namespace   = local.namespace
     environment = local.environment
     stage       = local.stage
-    attributes  = lower(replace(join(local.delimiter, local.attributes), local.regex_replace_chars, local.replacement))
+    attributes  = local.id_case == "none" ? replace(join(local.delimiter, local.attributes), local.regex_replace_chars, local.replacement) : lower(replace(join(local.delimiter, local.attributes), local.regex_replace_chars, local.replacement))
```

Comment:
@osterman As with `namespace` etc., I think we should do case conversion here for all cases.